platformio_esp32_envs.ini

```diff
@@ -22,6 +22,9 @@ build_flags               = ${mqtt_flags.build_flags}
                             -DCONFIG_LWIP_ESP_GRATUITOUS_ARP
                             -DCONFIG_LWIP_GARP_TMR_INTERVAL=30
 monitor_filters           = esp32_exception_decoder
+;upload_port               = COM4
+upload_speed              = 921600
```

Comment:
Does that work with all ESP32 boards/serial chips used?

Reply:
Oh sorry, I didn't want to check this in.

Reply:
I removed it.